modules/booklist.js

```diff
@@ -0,0 +1,24 @@
+// Retrieve saved book collection from localStorage, if available
+const bookCollection = JSON.parse(localStorage.getItem('bookCollection')) || [];
+// Function to render the book list
+export default function renderBookList() {
+  const bookListElement = document.getElementById('bookListItems');
+  bookListElement.innerHTML = '';
+  bookCollection.forEach((book, index) => {
+    const li = document.createElement('li');
+    li.textContent = `${book.title} by ${book.author}`;
+    li.className = 'single-book';
+    // Function to remove a book from the collection
+    function removeBook() {
```

Comment:
- Hey, please, kindly use the ES6 arrow function since you're working on learning to use the ES6 syntax.